Samsung/LPVS
# Put frontend assets to the lpvs container

# Description

This PR is intended to provide frontend and backend services in the lpvs container.

## Type of change


# How Has This Been Tested?
1. Update environment variables of `docker-compose.yml` file.
2. Build the lpvs container : `docker compose build`
3. Run containers: `docker compose up -d`
4. See the lpvs home page with any browser: `http://localhost:7896`

**Test Configuration**:
* Java: v11
* LPVS Release: v1.x.x

# Checklist:
